assignment09/part02/index.php

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<head>
+    <title>Assignment 9, Part 2</title>
+</head>
+<body>
+    
+    <?php
+        //Task 3.1: Change these items to reflect
+        //four of your favorite things
+        $my_favorite_things = array (
+            "My first favorite thing is singing",
+            "My second favorite thing is cooking",
+            "My third favorite thing is dancing",
+            "My fourth favorite thing is movie"
+        );
+    ?>
+
+    <h1>My Favorite Things</h1>
+        <!-- Task 3.2: Code to output things below -->
+        <?php 
+        $counter = 0
+        while ($counter < count($my_favorite_things)){
+            echo $my_favorite_things[$counter];
```

Comment:
nice while loop! You are outputting the elements of your array - but we're missing the ordered list (<ol></ol>). I'd expect the list to start and end before and after your loop and then have an <li> created for each element inside your loop. Let me know if you have questions.